src/scripts/popup.js

```diff
@@ -502,7 +500,8 @@ window.PopUp = {
       start = PopUp.getStart();
 
     if (start) {
-      request.start = start;
+      request.start = start.toISOString();
+      request.duration = -1 * Math.floor(start.getTime() / 1000);
```

Comment:
what do you base this value on? :thinking:

Reply:
https://github.com/toggl/toggl_api_docs/blob/master/chapters/time_entries.md

> duration: time entry duration in seconds. If the time entry is currently running, the duration attribute contains a negative value, denoting the start of the time entry in seconds since epoch (Jan 1 1970). The correct duration can be calculated as current_time + duration, where current_time is the current time in seconds since epoch. (integer, required)

Reply:
For currently running time entries the duration needs to be set to negative start timestamp. This was not done before, so the duration never gets updated and toggl-button uses the old duration timestamp on the stop event which results in entries that can read 

_start: 10:42, stop 10:53, duration 11 seconds_

Reply:
great, thanks for educating me :smile: